console/src/views/system/Setup.vue

```diff
@@ -1,207 +1,130 @@
-    // Create menu and menu items
-    const menuItemPromises = menuItems.map((item) => {
-      return apiClient.extension.menuItem.createv1alpha1MenuItem({
-        menuItem: item,
-      });
-    });
-    await Promise.all(menuItemPromises);
-    await apiClient.extension.menu.createv1alpha1Menu({ menu: menu });
-
-    // Install preset plugins
-    const { data: presetPlugins } = await apiClient.plugin.listPluginPresets();
-
-    for (let i = 0; i < presetPlugins.length; i++) {
-      pluginInstallMutate(presetPlugins[i]);
-    }
-  } catch (error) {
-    console.error("Failed to initialize preset data", error);
-  }
-
-  // Create system-states ConfigMap
-  await apiClient.extension.configMap.createv1alpha1ConfigMap({
-    configMap: {
-      metadata: {
-        name: "system-states",
-      },
-      kind: "ConfigMap",
-      apiVersion: "v1alpha1",
-      data: {
-        states: JSON.stringify({ isSetup: true }),
-      },
-    },
+  await apiClient.system.initialize({
+    systemInitializationRequest: formState.value,
   });
 
-  const systemStateStore = useSystemStatesStore();
-  await systemStateStore.fetchSystemStates();
-  const themeStore = useThemeStore();
-  await themeStore.fetchActivatedTheme();
+  const globalInfoStore = useGlobalInfoStore();
+  await globalInfoStore.fetchGlobalInfo();
 
   loading.value = false;
 
-  router.push({ name: "Dashboard" });
+  router.push({ name: "Login" });
 
   Toast.success(t("core.setup.operations.submit.toast_success"));
 };
 
-onMounted(async () => {
-  const systemStatesStore = useSystemStatesStore();
-
-  if (systemStatesStore.states.isSetup) {
-    router.push({ name: "Dashboard" });
-    return;
-  }
-
-  handleFetchSettings();
-  handleFetchConfigMap();
-});
+const inputClasses = {
+  outer: "!py-3 first:!pt-0 last:!pb-0",
+};
 </script>
 
 <template>
-  <div class="flex h-screen flex-col items-center justify-center">
+  <div class="flex h-screen flex-col items-center bg-white/90 pt-[30vh]">
     <IconLogo class="mb-8" />
-    <div class="flex w-72 flex-col gap-4">
+    <div class="flex w-72 flex-col">
       <FormKit
         id="setup-form"
+        v-model="formState"
         name="setup-form"
         :actions="false"
+        :classes="{
+          form: '!divide-none',
+        }"
+        :config="{ validationVisibility: 'submit' }"
         type="form"
         @submit="handleSubmit"
         @keyup.enter="$formkit.submit('setup-form')"
       >
         <FormKit
-          v-model="siteTitle"
-          :validation-messages="{
-            required: $t('core.setup.fields.site_title.validation'),
-          }"
+          name="siteTitle"
+          :classes="inputClasses"
+          :autofocus="true"
           type="text"
-          :placeholder="$t('core.setup.fields.site_title.placeholder')"
+          :validation-label="$t('core.setup.fields.site_title.label')"
+          :placeholder="$t('core.setup.fields.site_title.label')"
           validation="required|length:0,100"
```

Comment:
```suggestion
          validation="required:trim|length:0,100"
```

建议改为 `required:trim` 防止用户输入空格标题